src/main/java/org/folio/dew/batch/acquisitions/edifact/CompositePOLineConverter.java

```diff
@@ -106,6 +106,11 @@ public int convertPOLine(CompositePoLine poLine, EDIStreamWriter writer, int cur
       writePrice(poLine.getCost().getPoLineEstimatedPrice(), writer);
     }
 
+    if (poLine.getCost().getListUnitPrice() != null) {
```

Comment:
we can have 2 prices for electronic and physical - they are stored in different fields, additionally it should be multiplied by quantity as AAB is total price but without discounts, taxes etc

Reply:
Thanks, new calculation implemented by using function from mod-orders https://github.com/folio-org/mod-orders/blob/97a0e829a3497f2a5014f860785c1c3bcf9e7421/src/main/java/org/folio/orders/utils/HelperUtils.java#L254